infer/models/c/src/libc_basic.c

```diff
@@ -682,6 +682,9 @@ unsigned sleep(unsigned seconds) {
 #ifdef __APPLE__ // define __WAIT_STATUS as int * on mac
 #define __WAIT_STATUS int*
 #endif
+#ifdef __GNUC__  // define __WAIT_STATUS as int * with glibc
```

Comment:
Yes, seems like they got rid of the constant (__WAIT_STATUS, by replacing with int* everywhere) as well: https://github.com/bminor/glibc/commit/b49ab5f4503f36dcbf43f821f817da66b2931fe6 

I will use the ifdef, thanks.